### ISSUE TITLE
Cannot install ros_kortex. unable to connect to kinova_public...

### ISSUE THREAD

**Issue by RaduCorcodel** [Kinovarobotics/ros_kortex#203]
## Description

Trying to install ros_kortex. Followed all installation steps and during the configuration step I get:

```
-- Conan: checking conan executable in path
-- Conan: Found program /usr/local/bin/conan
-- Conan: Version found Conan version 1.40.1

-- Conan: Adding kinova_public remote repository (https://artifactory.kinovaapps.com/artifactory/api/conan/conan-public)
-- Conan: Automatic detection of conan settings from cmake
-- Conan: Settings= -s;build_type=Release;-s;compiler=gcc;-s;compiler.version=9;-s;compiler.libcxx=libstdc++11;-s;kortex_api_cpp:compiler=gcc;-s;kortex_api_cpp:compiler.version=5;-s;kortex_api_cpp:compiler.libcxx=libstdc++11
-- Conan executing: conan install /homes/robodemo/ws_kinova/src/ros_kortex/kortex_driver/conanfile.py -s build_type=Release -s compiler=gcc -s compiler.version=9 -s compiler.libcxx=libstdc++11 -s kortex_api_cpp:compiler=gcc -s kortex_api_cpp:compiler.version=5 -s kortex_api_cpp:compiler.libcxx=libstdc++11 -g=cmake --update
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=9
os=Linux
os_build=Linux
kortex_api_cpp:compiler=gcc
kortex_api_cpp:compiler.version=5
kortex_api_cpp:compiler.libcxx=libstdc++11
[options]
[build_requires]
[env]

ERROR: Failed requirement 'kortex_api_cpp/2.3.0-r.34@kortex/stable' from 'conanfile.py'
ERROR: HTTPSConnectionPool(host='artifactory.kinovaapps.com', port=443): Max retries exceeded with url: /artifactory/api/conan/conan-public/v1/ping (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1131)')))

Unable to connect to kinova_public=https://artifactory.kinovaapps.com/artifactory/api/conan/conan-public
1. Make sure the remote is reachable or,
2. Disable it by using conan remote disable,
Then try again.
CMake Error at ros_kortex/kortex_driver/cmake/conan.cmake:399 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  ros_kortex/kortex_driver/cmake/conan.cmake:485 (conan_cmake_install)
  ros_kortex/kortex_driver/CMakeLists.txt:72 (conan_cmake_run)


-- Configuring incomplete, errors occurred!
```

## Version

ROS distribution :  ros-noetic

Branch and commit you are using : d53b135

Any suggestion is welcomed.

**Comment by felixmaisonneuve** [comment]
Hi @RaduCorcodel,

See this issue I created : #198 

You can fix this by using 
```
conan config install https://github.com/conan-io/conanclientcert.git

```

Regards,
Felix

**Comment by RaduCorcodel** [comment]
Hi @felixmaisonneuve,

That did the trick. Thank you. I assume this issue will be resolved in #198 so we can merge these together or close this issue, whichever you think is best.

Regards,
Radu

**Comment by felixmaisonneuve** [comment]
Yeah I wanted to let it open for a bit for people to see it, but I'll make sure to close it at some point.
